uk.co.compucorp.civicrm.tasksassignments/api/v3/Document.php

```diff
@@ -232,32 +214,43 @@ function civicrm_api3_document_create_multiple($params) {
  */
 function _civicrm_api3_document_create_spec(&$params) {
 
-  //default for source_contact_id = currently logged in user
-  $params['source_contact_id']['api.default'] = 'user_contact_id';
```

Comment:
I removed it because it's duplicated in a block a few lines below `$params['source_contact_id'] = [`